src/seedu/addressbook/data/person/ReadOnlyPerson.java

```diff
@@ -82,4 +82,16 @@ default String getAsTextHidePrivate() {
         }
         return builder.toString();
     }
+
+    /**
+     * Returns a concatenated version of the printable strings of each object.
+     */
+    default String getPrintableString(Printable... printables){
```

Comment:
Good that you remembered to give header comments for this method!

Ideally, the new method `getPrintableString` needs to be used somewhere.